### PR TITLE
Added required sections in sympy.simplify

### PR DESCRIPTION
**Pull request by Soumi7**
# Updates sympy.simplify to follow latest docstring style guide

#### References to other Issues or PRs

#19591

#### Brief description of what is fixed or changed
- Required sections in :
  - combsimp.py
  - cse_main.py
  - epathtools.py 
  - fu.py
  - gammasimp.py 
  - hyperexpand.py
  - powsimp.py
  - radsimp.py

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/simplify/fu.py [hunk 1]
```diff
@@ -1623,8 +1623,8 @@ def fu(rv, measure=lambda x: (L(x), x.count_ops())):
 
     References
     ==========
-    http://rfdz.ph-noe.ac.at/fileadmin/Mathematik_Uploads/ACDCA/
-    DESTIME2006/DES_contribs/Fu/simplification.pdf
+
+    .. [1] http://rfdz.ph-noe.ac.at/fileadmin/Mathematik_Uploads/ACDCA/DESTIME2006/DES_contribs/Fu/simplification.pdf
```
Comment (sylee957) [inline sympy/simplify/fu.py:1627]: This URL is not available. I may change this reference to
https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.657.2478&rep=rep1&type=pdf

File: sympy/simplify/radsimp.py [hunk 2]
```diff
@@ -989,7 +995,7 @@ def handle(expr):
 
 def rad_rationalize(num, den):
     """
-    Rationalize num/den by removing square roots in the denominator;
+    Rationalize ``num``/``den`` by removing square roots in the denominator;
```
Comment (sylee957) [inline sympy/simplify/radsimp.py:998]: This is already the pythonic expression, so it should better be
```
``num/den``
```

Comment (Soumi7) [reply sympy/simplify/radsimp.py]: Changed.

File: sympy/simplify/fu.py (hunk 1 above)
Comment (Soumi7) [reply sympy/simplify/fu.py]: Changed the URL to the one mentioned above.

**Comment by sylee957** [comment]
I can’t merge this right now because the unrelated tests are failing, but I approve that anyone can merge after the master is fixed.

**Pull request merged**